src/actions/commands/commandLine.ts

```diff
@@ -259,6 +259,8 @@ class CommandInsertInSearchMode extends BaseCommand {
     ['<C-n>'], // Next
     ['<C-f>'], // Find
     ['<C-u>'], // Delete to beginning
+    // Another way to run search
+    ['<C-m>'],
```

Comment:
nit: can you put the comment on the same line to follow the pattern established above?